src/test/java/core/basesyntax/service/reports/ReportServiceImpTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.service.reports;
+
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.FruitStorage;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportServiceImpTest {
+    private static ReportServiceImp reportServiceImp;
+    private static String header;
+    private static Map<String, Integer> storage;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        reportServiceImp = new ReportServiceImp(new FruitDaoImpl());
+        header = "fruit,quantity";
+        storage = FruitStorage.storage;
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        storage.clear();
+    }
+
+    @Test
+    public void reportService_validStore_ok() {
```

Comment:
```suggestion
    public void create_validStore_ok() {
``` 
the same for the method below